### ISSUE TITLE
Problems with adding phone number

### ISSUE THREAD

**Issue by twisterdw** [keybase/client#20345]
I already add my 2 phone numbers and confirm it with code from sms but when i try to check myself to airdrop program app says that i must to add a phone number in settings.can someone help me with that?

**Comment by bitjas** [comment]
Yeah I have the same problem , trying to add my phone number but it never sends me the sms code to confirm :(

**Comment by heronhaye** [comment]
@twisterdw @bitjas what country is your phone number from? and what carrier?

**Comment by twisterdw** [comment]
Yes hello.i am from georgia and have georgian number Geocell

**Comment by heronhaye** [comment]
@twisterdw Can you screenshot this from the app?
>i must to add a phone number in settings.

**Comment by bitjas** [comment]
Hi, I from new Zealand and my carrier is Vodafone.
 The site told me I was eligible for the Stella airdrop, then you changed
the rules then I missed out cause I couldn't confirm my phone number.

**Comment by twisterdw** [comment]
Yes sure.
